torq.q

```diff
@@ -610,14 +611,18 @@ if[@[value;`.ps.loaded;0b]; .ps.initialise[]]
 // initialise connections
 if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
+.proc.run:{[id;a]
+    .lg.o[id;"attemping to run: ",.Q.s1 a];
+    @[value;a;
+    {[id;x;a].lg.e[id;x," error - failed to run: ",.Q.s1 a]}[id;;a]];
```

Comment:
similar to prior comment from @ThomasSmyth I would rearrange args here to `[id;a;x]` so you can remove a `;` in `[id;;a]` after lambda